packages/react-composites/src/composites/MeetingComposite/adapter/AzureCommunicationMeetingAdapter.ts

```diff
@@ -395,7 +398,9 @@ export class AzureCommunicationMeetingAdapter implements MeetingAdapter {
  */
 export interface CallAndChatLocator {
   /** Locator used by {@link createAzureCommunicationMeetingAdapter} to locate the call to join */
-  callLocator: GroupCallLocator;
+  callLocator:
+    | GroupCallLocator
+    | /* @conditional-compile-remove-from(stable) TEAMS_ADHOC_CALLING */ CallParticipantLocator;
```

Comment:
As noted before, everything under `MeetingComposite` is beta only, so no need for conditional compilation here.

OTOH, this is related to a different feature -- and we may not stabilize TEAMS_ADHOC_CALLING at the same time.

So I reverse my vote -- keep this conditional compilation directive.

Reply:
Yeah different feature (so when meetings composite goes stable this will still be beta)